core/01_core.js

```diff
@@ -694,8 +705,17 @@
       unhandledPromiseRejectionHandler = handler,
     reportUnhandledException: (e) => op_dispatch_exception(e, false),
     reportUnhandledPromiseRejection: (e) => op_dispatch_exception(e, true),
-    queueTimer: (depth, repeat, timeout, task) =>
-      op_timer_queue(depth, repeat, timeout, task),
+    queueUserTimer: (depth, repeat, timeout, task) => {
+      const id = op_timer_queue(depth, repeat, timeout, task);
+      if (__isLeakTracingEnabled()) {
+        submitTimerTrace(id);
+      }
+      return id;
+    },
+    // TODO(mmastrac): Hook up associatedOp to tracing
```

Comment:
Future work, implementing a stub here for now so we can get ready for it in Deno.